telemetry/definitions/commonDefinitions.json

```diff
@@ -543,6 +543,12 @@
             "type": "string",
             "description": "The type of the user trigger to send request to CodeWhisperer service",
             "allowedValues": ["OnDemand", "AutoTrigger"]
+        },
+        {
+            "name": "syncedResources",
+            "type": "string",
+            "description": "Describes which parts of an application (that we know of) were synced to the cloud. \"Code\" resources follow the SAM spec: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-sync.html",
+            "allowedValues": ["Everything", "CodeOnly"]
```

Comment:
Could this value be changed to AllResources?

Reply:
Sure